q2_gneiss/plot/tests/test_plot.py

```diff
@@ -347,32 +347,23 @@ def test_balance_taxonomy_partial_numerical_categorical(self):
         balance_taxonomy(self.results, self.table, self.tree,
                          self.taxonomy, balance_name='a',
                          metadata=self.partial_numerical_categorical)
-        self.assertTrue(os.path.exists(index_fp))
-        # test to make sure that the numerator file is there
-        num_fp = os.path.join(self.results, 'numerator.csv')
-        self.assertTrue(os.path.exists(num_fp))
-        # test to make sure that the denominator file is there
-        denom_fp = os.path.join(self.results, 'denominator.csv')
-        self.assertTrue(os.path.exists(denom_fp))
-        box_fp = os.path.join(self.results, 'balance_metadata.pdf')
-        self.assertTrue(os.path.exists(box_fp))
-        prop_fp = os.path.join(self.results, 'proportion_plot.pdf')
-        self.assertFalse(os.path.exists(prop_fp))
 
-        box_fp = os.path.join(self.results, 'balance_metadata.pdf')
-        self.assertTrue(os.path.exists(box_fp))
-        prop_fp = os.path.join(self.results, 'proportion_plot.pdf')
-        self.assertFalse(os.path.exists(prop_fp))
+        self.assertTrue(os.path.exists(index_fp))
```

Comment:
Why is this lil fella being all lonely up here? Can't it be moved downstairs into the loop party, too?

Reply:
Because the loop joins the file name to the current path in the assertion, but that `index_fp` variable already is the filename joined to the current path (and with good reason because that file is actually opened later), so I figured I would validate the already existing path instead of rejoining the filename `index.html` to the path in a throwaway variable in the loop.

Reply:
No need to include this assertion if [L358](https://github.com/qiime2/q2-gneiss/pull/60/files#diff-7c88d32ba03a1f712d611282202df837R358) is actually opening the file up and interrogating it.